regression/cbmc/r_w_ok_inconsistent_integer/test-global.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+main.c
+--function test_global_pointer --pointer-check
+^EXIT=0$
```

Comment:
Please check verification failed and that the assert fails :tophat:

Reply:
This test should succeed :rabbit: